### PR TITLE
Insectoids (Roachmen) [WIP] [DNM]

### PR DESCRIPTION
**Pull request by Buchse**
Here's what I've got so far working on my insect race. I've gotten everything to compile and run with no errors, they show up correctly in the character preferences preview. On top of this I'm able to pick and spawn both they and their monkey versions using the create mob button. 

The only issue that seems to remain is that upon spawning, either roundstart or forced, they spawn completely invisible. VVing them shows that they have the correct sprites selected, but they just don't seem to show up.
[wip] [dnm]
:cl:
 * rscadd: Added in sprites for Insectoids, created a proc that ignores radiation, added the Insectoid language, added a monkey subtype for Insectoids.
 * rscadd: Added a proper organ list, eye var, and fixed an issue in human.dm with a missing "/"

### CODE REVIEW

File: code/modules/mob/living/carbon/species.dm [hunk 1]
```diff
@@ -1281,6 +1281,34 @@ var/list/has_died_as_golem = list()
...
+	tox_mod = 0.5
+
+	blood_color = "#e6e600"
+	flesh_color = "#9c7f25"
+
+	has_mutant_race = 0
+
+	species_intro = "You are an Insectoid.<br>\
+					Your body is utterly immune to the perils of radiation, and you are able to better defend against toxic chemicals <br>\
+					However, your rigid body is somewhat more fragile than that of more soft-bodied species. Resilient though you may be, a good smack may put you out of comission."
```
Comment (kilozombie) [inline code/modules/mob/living/carbon/species.dm]: `commission` here

File: code/modules/mob/living/carbon/species.dm [hunk 2]
```diff
@@ -1281,6 +1281,34 @@ var/list/has_died_as_golem = list()
...
+
+/datum/species/skrell/gib(mob/living/carbon/human/H) //placeholder til I get this figured
```
Comment (kilozombie) [inline code/modules/mob/living/carbon/species.dm]: Might be an accidental copypaste botch, did you mean `/datum/species/insectoid/gib`?

Comment (kilozombie) [reply code/modules/mob/living/carbon/species.dm]: I ask because I did something similar and it was the reason Lamprey kept crashing for months on end

**Comment by calzilla1** [comment]
Waiting for someone to ERP with this race
https://youtu.be/uubHtO21J_s

File: code/modules/mob/living/carbon/species.dm (hunk 2 above)
Comment (Buchse) [reply code/modules/mob/living/carbon/species.dm]: no, the gib animation's a WIP, I'm just stumped as to actual causes

**Comment by ArchSunder** [comment]
call them Tianan Men

File: code/modules/mob/living/carbon/monkey/roach.dm [hunk 3]
```diff
@@ -0,0 +1,21 @@
+/mob/living/carbon/monkey/roach
+	name = "roach"
+	voice_name = "roach"
+	speak_emote = list("hisses")
+	icon_state = "bigroach"
+	meat_type = /obj/item/weapon/reagent_containers/food/snacks/meat/roach/big
+	species_type = /mob/living/carbon/monkey/rock
```
Comment (MadmanMartian) [inline code/modules/mob/living/carbon/monkey/roach.dm]: ???

Comment (Buchse) [reply code/modules/mob/living/carbon/monkey/roach.dm]: ah, thats a good catch, i based their code on the golem monkies

**Pull request closed without merging**